distributions/__init__.py

```diff
@@ -60,6 +61,7 @@
 "Dist_sm6779",
 "Dist_yz4244",
 "Dist_speedreed2",
+"Dist_yl6709"
```

Comment:
You will want to do this in a different branch for testing. If I merge this it will cause conflicts for others.
Can you remove this line.

Reply:
Oh sorry I missed the remark! I have removed the two lines I added in this file. Please see if this works now.